test/addon_selection_dialog_test.rb

```diff
@@ -70,6 +64,52 @@
       expect(wrapped_addon.selected?).to eq true
     end
 
+    it "preselects the recommended addons" do
+      addon = addon_generator("recommended" => true)
+
+      expect(Yast::UI).to receive(:UserInput).and_return(:next)
```

Comment:
These three lines are pretty much the same in both tests. I would consider then as preconditions so maybe you could move them to a `before` block tunning them using `let` calls.